storage/src/vespa/storage/persistence/bucketownershipnotifier.cpp

```diff
@@ -85,7 +85,7 @@ BucketOwnershipNotifier::logNotification(const document::BucketId& bucket,
         "%s now owned by distributor %u, but reply for operation is scheduled "
         "to go to distributor %u. Sending NotifyBucketChange with %s to ensure "
         "new owner knows bucket exists",
-        bucket.toString().c_str(),
+        bucket.getBucketId().toString().c_str(),
```

Comment:
For log messages it's perfectly fine to do `toString()` on the top-level `Bucket` itself instead of just the `BucketId`. We're likely to want the bucket space logged for debugging purposes soon either way.